### PR TITLE
Offboard mailing address outside of NY 

### PR DESCRIPTION
**Pull request by tahsinaislam**

### CODE REVIEW

**Review by loumoore — CHANGES_REQUESTED** [review]
left one maybe-blocking comment. the rest looks good tho!

File: app/lib/navigation/state_file_ny_question_navigation.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@ class StateFileNyQuestionNavigation
       StateFile::Questions::EligibilityResidenceController,
       StateFile::Questions::EligibilityOutOfStateIncomeController,
       StateFile::Questions::NyEligibilityCollegeSavingsWithdrawalController,
-      StateFile::Questions::EligibilityOffboardingController,
```
[the remark this answers is not in the document]
Comment (tahsinaislam) [reply app/lib/navigation/state_file_ny_question_navigation.rb:10]: The logic right now in this controller still off boards people right away once they answer any one of the earlier questions with offboarding criteria

[the remark this answers is not in the document]
Comment (loumoore) [reply app/lib/navigation/state_file_ny_question_navigation.rb:10]: gotcha, cool. i see there is a concern that the controller's that might result in offboarding can implement, which takes the user to this controller when it should 👍  i'm curious though, does it even need to be in the navigation flow here? why did you need to move it around? thanks!

File: app/forms/state_file/ny_permanent_address_form.rb [hunk 2]
```diff
@@ -30,6 +32,7 @@ def save
                                   permanent_street: @intake.direct_file_data.mailing_street,
                                   permanent_zip: @intake.direct_file_data.mailing_zip,
                                 } : {}
+      attributes_from_direct_file[:permanent_address_outside_ny] = @intake.direct_file_data.mailing_state != 'NY' && confirmed_permanent_address == "yes" ? "yes" : "no"
```
Comment (tahsinaislam) [inline app/forms/state_file/ny_permanent_address_form.rb:35]: @bengolder does this look ok?

File: app/lib/navigation/state_file_ny_question_navigation.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (tahsinaislam) [reply app/lib/navigation/state_file_ny_question_navigation.rb]: Fixed this with Ben!

**Review by loumoore — APPROVED** [review]

**Pull request merged**